### PR TITLE
Audio : Fix Profiler Bug + Added stops events fade

### PR DESCRIPTION
**Pull request by AsoboStudio**
audio - added a fix profiler bug + added Stops events with fades + if emitter is null, UAudioManager = emitter

### CODE REVIEW

File: Assets/HoloToolkit/SpatialSound/Scripts/UAudioManager/UAudioManager.cs [hunk 1]
```diff
@@ -54,12 +54,12 @@ protected new void Awake()
 
             CreateEventsDictionary();
 
-            if (events.Length > 0)
-            {
-                string key = events[0].name;
-                PlayEvent(key);
-                StopEvent(key);
-            }
+            //if (events.Length > 0)
```
Comment (NeerajW) [inline Assets/HoloToolkit/SpatialSound/Scripts/UAudioManager/UAudioManager.cs]: Are these still needed? If not then please delete.

Comment (manudoulut) [inline Assets/HoloToolkit/SpatialSound/Scripts/UAudioManager/UAudioManager.cs]: Yes , it seems that we don't need those lines. We will delete it.

Comment (NeerajW) [inline Assets/HoloToolkit/SpatialSound/Scripts/UAudioManager/UAudioManager.cs]: Please update the PR and I will merge it. Thanks!

**Comment by AsoboStudio** [comment]
Removed needless lines

**Pull request merged**